app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section id='shopping-list' class="container mx-auto px-4">
```

Comment:
- [ ] Good job so far but I have an issue with your design. I noticed that your shopping list doesn't follow the project wireframe. Please fix it in order to make your app design follows the wireframe 👍 

#### Your design
![general_shopping](https://github.com/Solobarine/recipes/assets/6615394/cc2128a7-66fd-44f2-8b2b-8157f7a8089c)

#### Wireframe
![general_shopping_list](https://github.com/Solobarine/recipes/assets/6615394/df6d4f54-2b3b-4cb2-99ca-92818a6d1345)